docs/providers/aws/guide/intro.md

```diff
@@ -79,7 +79,93 @@ functions: # Your "Functions"
 resources: # The "Resources" your "Functions" use.  Raw AWS CloudFormation goes in here.
 ```
 
-When you deploy with the Framework by running `serverless deploy`, everything in `serverless.yml` is deployed at once.
+_serverless.json_
+
+```json
+{
+  "service": "users",
+  "functions": {
+    "usersCreate": {
+      "events": [
+        {
+          "http": "post users/create"
+        }
+      ]
+    },
+    "usersDelete": {
+      "events": [
+        {
+          "http": "delete users/delete"
+        }
+      ]
+    }
+  },
+  "resources": {}
+}
+```
+
+The Framework also handles natively exporting JSON object from a Javascript file `serverless.js` or from a Typescript file `serverless.ts` at the root directory of the project. While the Framework is language-agnostic, projects written in Node.js will highly benefit from using the same language for the service definition file. Such service file looks like this:
+
+_serverless.js_
+
+```js
+'use strict';
+
+module.exports = {
+  service: 'users',
+  functions: {
+    usersCreate: {
+      events: [
+        {
+          http: 'post users/create',
+        },
+      ],
+    },
+    usersDelete: {
+      events: [
+        {
+          http: 'delete users/delete',
+        },
+      ],
+    },
+  },
+  resources: {},
+};
+```
+
+_serverless.ts_
+
+```ts
+// Requiring @types/serverless in your project package.json
+import { Serverless } from 'serverless/aws';
```

Comment:
Is `serverless/aws` path correct? To me it looks as it leads to not existing resource

Reply:
The definitelyTyped librairy has an `aws.d.ts` at the root of the lib exporting AWS from `./plugins/aws/provider/awsProvider` -> https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/serverless/aws.d.ts

Reply:
Ok, so that path is not resolved by Node.js rules, that's quite confusing (as regular ESM import is used)